ush/templates/FV3LAM_wflow.xml

```diff
@@ -321,32 +334,42 @@ MODULES_RUN_TASK_FP script.
     <nodes>{{ nnodes_get_extrn_ics }}:ppn={{ ppn_get_extrn_ics }}</nodes>
     <walltime>{{ wtime_get_extrn_ics }}</walltime>
     <nodesize>&NCORES_PER_NODE;</nodesize>
-    <jobname>&TAG;_&GET_EXTRN_ICS_TN;</jobname>
-    <join><cyclestr>&LOGDIR;/&GET_EXTRN_ICS_TN;_@Y@m@d@H.log</cyclestr></join>
+    <jobname>&TAG;_&GET_EXTRN_ICS_TN;{{ uscore_ensmem_name }}</jobname>
+    <join><cyclestr>&LOGDIR;/&GET_EXTRN_ICS_TN;{{ uscore_ensmem_name }}_@Y@m@d@H.log</cyclestr></join>
 
     <envar><name>GLOBAL_VAR_DEFNS_FP</name><value>&GLOBAL_VAR_DEFNS_FP;</value></envar>
     <envar><name>PDY</name><value><cyclestr>@Y@m@d</cyclestr></value></envar>
     <envar><name>CDATE</name><value><cyclestr>@Y@m@d@H</cyclestr></value></envar>
     <envar><name>CYCLE_DIR</name><value><cyclestr>&CYCLE_BASEDIR;/@Y@m@d@H</cyclestr></value></envar>
     <envar><name>EXTRN_MDL_NAME</name><value>{{ extrn_mdl_name_ics }}</value></envar>
     <envar><name>ICS_OR_LBCS</name><value>ICS</value></envar>
+    <envar><name>SLASH_ENSMEM_SUBDIR</name><value>{{ slash_ensmem_subdir }}</value></envar>
+    <envar><name>INPUT_ENSMEM_SUBDIR</name><value>#subdirGE#</value></envar>
+    <envar><name>POST_ENSMEM_SUBDIR</name><value>/postprd_#memNameWRF#</value></envar>
+    <envar><name>ENSMEM_NAME</name><value>#memNameWRF#</value></envar>
```

Comment:
I think it is fine to have extra envars as long as it does not break anything. But I think it will be best if you can merge "POST_ENSMEM_SUBDIR" and "ENSMEM_NAME" into one variable since if you know memNameWRF, you know how to construct POST_ENSMEM_SUBDIR in the script.  Also it helps if you make the variable name more descriptive, for example, GEFS_INPUT_SUBDIR,  WRF_MEM_NAME.